src/kraken/core/system/executor/default.py

```diff
@@ -151,6 +152,14 @@ def after_execute_graph(self, graph: Graph) -> None:
                     self.status_to_text(status),
                     self.format_duration(f"[{self._duration[task_path]:.3f}s]") if task_path in self._duration else "",
                 )
+        not_executed_tasks = list(graph.tasks(not_executed=True))
+        if len(not_executed_tasks) != 0:
+            print(flush=True)
+            print(self.format_header("Tasks that were not executed due to failing dependencies"), flush=True)
+            print(flush=True)
+            for task in not_executed_tasks:  # Keeping only the pending tasks
+                print(" " * (len(self.execute_prefix) + 1) + task.path)  # path seems to be
+                # Deprecated in v0.13.0 (task.py l231)
```

Comment:
- `# Keeping only the pending tasks` should no longer be here
  - it should be above, but in fact I'd argue it should not exist at all, it is just paraphrasing the function call `not_executed_tasks = list(graph.tasks(not_executed=True))` :)
- `path` is indeed deprecated, but IMO the comment is not adding any value

```suggestion
            for task in not_executed_tasks:
                print(" " * (len(self.execute_prefix) + 1) + task.path)
```